tests/functions/test_strings.py

```diff
@@ -8,10 +8,22 @@
 from data_safe_haven.functions import next_occurrence, sanitise_sre_name
 
 
-def test_next_occurrence_is_within_next_day():
+def test_next_occurrence_will_occur_soon():
     dt_utc_now = datetime.datetime.now(datetime.UTC)
     next_time = next_occurrence(5, 13, "Australia/Perth")
-    dt_utc_tomorrow = dt_utc_now + datetime.timedelta(days=1, minutes=5)
+    dt_utc_tomorrow = dt_utc_now + datetime.timedelta(days=1, hours=2)
```

Comment:
If you ran this at 23:00 wouldn't `dt_utc_tomorrow` be the day after tomorrow?

I think it is probably better to parameterise the tests over some set values, including edge cases near day boundaries.
That way the results and catching bugs doesn't depend on when the tests are run.

Reply:
I guess that would require Mock-ing `datetime.datetime.now()` inside the test? Or is there another way?

Reply:
Yes that sounds right.

I think the clever way to do it would be to make a fixture to mock `datetime.datetime.now()` which takes an argument that is the datetime to return. Then you could loop over that with `pytest.parametrize`.

Reply:
Done with `freezegun` in 09af7930c